gsad/src/gsad_gmp.c

```diff
@@ -1124,209 +908,74 @@ get_one (gvm_connection_t *connection, const char *type,
  *
  * @return Enveloped XML object.
  */
-static char *
-get_many (gvm_connection_t *connection, const char *type,
-          credentials_t *credentials, params_t *params, const char *extra_xml,
-          const char *extra_attribs, cmd_response_data_t *response_data)
+char *
+get_one (gvm_connection_t *connection, const char *type,
+         credentials_t *credentials, params_t *params, const char *extra_xml,
+         gmp_arguments_t *arguments, cmd_response_data_t *response_data)
```

Comment:
The documentation comment of get_one still refers to extra_attribs instead of arguments.

Reply:
Fixed